src/renderer/reader/redux/sagas/resourceCache.ts

```diff
@@ -0,0 +1,89 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { IReaderRootState } from "readium-desktop/common/redux/states/renderer/readerRootState";
+import { ICacheDocument } from "readium-desktop/common/redux/states/renderer/resourceCache";
+import { ContentType } from "readium-desktop/utils/contentType";
+import {
+    all as allTyped, select as selectTyped,
+    call as callTyped, put as putTyped,
+} from "typed-redux-saga/macro";
+import { Publication as R2Publication } from "@r2-shared-js/models/publication";
+import { Link } from "@r2-shared-js/models/publication-link";
+
+import * as debug_ from "debug";
+import { readerLocalActionSetResourceToCache } from "../actions";
+const debug = debug_("readium-desktop:renderer:reader:redux:saga:resourceCache");
+
+const isFixedLayout = (link: Link, publication: R2Publication): boolean => {
+    if (link && link.Properties) {
+        if (link.Properties.Layout === "fixed") {
+            return true;
+        }
+        if (typeof link.Properties.Layout !== "undefined") {
+            return false;
+        }
+    }
+
+    if (publication &&
+        publication.Metadata &&
+        publication.Metadata.Rendition) {
+        return publication.Metadata.Rendition.Layout === "fixed";
+    }
+    return false;
+};
+
+export function* getResourceCache() {
+
+    const cacheFromState = yield* selectTyped((state: IReaderRootState) => state.resourceCache);
+
+    // TODO: check this before merge
```

Comment:
need to double check this

Reply:
make sure this is not persisted in local Redux disk snapshot, right? (heavy data set)

Reply:
It's not persisted, only state.reader is persisted